librz/core/cmd/cmd_print.c

```diff
@@ -6570,3 +6474,82 @@ RZ_IPI RzCmdStatus rz_print_value4_handler(RzCore *core, int argc, const char **
 RZ_IPI RzCmdStatus rz_print_value8_handler(RzCore *core, int argc, const char **argv, RzCmdStateOutput *state) {
 	return print_value_size(core, state, argc, argv, 8);
 }
+
+static void printraw(RzCore *core, int len) {
+	int obsz = core->blocksize;
+	int restore_obsz = 0;
+	if (len != obsz) {
+		if (!rz_core_block_size(core, len)) {
+			len = core->blocksize;
+		} else {
+			restore_obsz = 1;
+		}
+	}
+	rz_print_raw(core->print, core->offset, core->block, len);
+	if (restore_obsz) {
+		(void)rz_core_block_size(core, obsz);
+	}
+	core->cons->newline = core->cmd_in_backticks ? false : true;
+}
```

Comment:
I think this whole function should be replaced by using rz_io functions directly instead of changing blocks back and forth. @thestr4ng3r cc

Reply:
Done